org.gnome.Geary.json

```diff
@@ -136,8 +169,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/evolution-data-server/3.32/evolution-data-server-3.32.3.tar.xz",
-                    "sha256": "6481f385dd9977b6d8442f356210921ac63244cb5a188c6986b72ebe37f7a2eb"
+                    "url": "https://download.gnome.org/sources/evolution-data-server/3.32/evolution-data-server-3.32.4.tar.xz",
```

Comment:
Any reason for not using 3.34?

Reply:
Yup, per the commit: 

> Keep EDS at 3.32 for the moment while most people have that installed on the host, and hence keep folks at 0.12.

Reply:
That's the short version anyway, the long version is we really need a contacts portal.

Reply:
I should have looked at the commit message first.